packages/pyright-internal/src/common/serviceProviderExtensions.ts

```diff
@@ -77,6 +77,10 @@ ServiceProvider.prototype.console = function () {
     return this.get(ServiceKeys.console);
 };
 ServiceProvider.prototype.partialStubs = function () {
+    const result = this.tryGet(ServiceKeys.partialStubs);
```

Comment:
Should we add a test?

Reply:
That would be a lot of work? There's no test for running pyright from the command line. There are tests for the language server (which didn't have this bug) and there's tests to make sure partial stub service works, but there's no easy way to have a test where we validate the CLI is behaving as expected. Other than the mypy_primer. Which looks like it's showing a difference.